target/linux/ramips/image/mt7621.mk

```diff
@@ -1109,7 +1109,7 @@ TARGET_DEVICES += netgear_r6900-v2
 
 define Device/netgear_r7200
   $(Device/netgear_sercomm_nand)
-  DEVICE_MODEL := R7200
+  DEVICE_MODEL := R7450
```

Comment:
That's wrong.

Reply:
https://github.com/openwrt/openwrt/pull/4408/files#diff-5ac2666a9f7c325aa6d42d94872a91ebe7d8af73e2f708dc0f8564b0f3f88cd8R1124 
Is also wrong, should be swapped?